lldb/source/Plugins/StructuredData/DarwinLog/StructuredDataDarwinLog.cpp

```diff
@@ -252,7 +253,8 @@ class FilterRule {
 
   const bool m_accept;
   const size_t m_attribute_index;
-  const ConstString m_operation;
+  // The lifetime of m_operation should be static.
```

Comment:
Static seems confusing in this context. Do you mean should be fixed for the lifetime of this object?

Reply:
Ah, yeah it is a little confusing now that I look at it. I wanted to convey that the lifetime of the StringRef should be as long as the lifetime of the program (since it's backed by a static string literal right now). Maybe something like:

"m_operation is backed by a static string literal, so lifetime should not be a concern"